tests/simulation/test_simulator.py

```diff
@@ -1,10 +1,14 @@
 import random
+import sys
 from typing import Optional
 
+import pytest
+
 from hathor.simulator import FakeConnection, MinerSimulator, RandomTransactionGenerator, Simulator
 from tests import unittest
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason='set_seed fails on Windows')
```

Comment:
What's the error message? Maybe I can get it fixed.

Reply:
I actually haven't debugged it at all. I don't know if it's worth the time, I think we can leave this to a separate task.

It has something to do with `numpy.random.set_seed`.

Reply:
Same as above. Thanks!